test/integration/org/transmart/dataexport/ClinicalExportServiceTests.groovy

```diff
@@ -90,35 +98,65 @@ class ClinicalExportServiceTests {
     }
     @Test
     void testWithConceptPathSpecified() {
-        def file = clinicalExportService.exportClinicalData(
+        def files = clinicalExportService.exportClinicalData(
                 jobName: 'test',
                 resultInstanceId: queryResult.id,
                 conceptKeys: [ sexNode.key.toString() ],
                 studyDir: tmpDir)
 
-        assertTrue(file.exists())
-        assertThat file.absolutePath, endsWith('/data_clinical.tsv')
-        assertThat file.length(), greaterThan(0l)
-        def table = file.text.split('\n')*.split('\t').collect { it as List }
-        assertThat table, contains(
+        assertThat files, containsInAnyOrder(
+                hasProperty('absolutePath', endsWith('/data_clinical.tsv')),
+                hasProperty('absolutePath', endsWith('/meta.tsv')),
+        )
+
+        files.each { file ->
+            assertTrue(file.exists())
+            assertThat file.length(), greaterThan(0l)
+        }
+
+        def dataFile = files.find { it.absolutePath.endsWith '/data_clinical.tsv' }
+
+        def dataTable = dataFile.text.split('\n')*.split('\t').collect { it as List }
+        assertThat dataTable, contains(
                 contains('"Subject ID"', '"\\foo\\study2\\sex\\"'),
                 contains('"SUBJ_ID_3"', '"female"'),
                 contains('"SUBJ_ID_2"', '"male"'),
                 contains('"SUBJ_ID_1"', '"female"'),
         )
+
+        def metaFile = files.find { it.absolutePath.endsWith '/meta.tsv' }
+
+        def metaTable = metaFile.text.split('\n')*.split('\t').collect { it as List }
+        assertThat metaTable, contains(
+                contains('"Variable"', '"Attribute"', '"Description"'),
+                contains('"\\foo\\study2\\sex\\"', '"8 name 2"', '"8 description 2"'),
+                contains('"\\foo\\study2\\sex\\"', '"8 name 1"', '"8 description 1"'),
+                contains('"\\foo\\study2\\sex\\female\\"', '"10 name 2"', '"10 description 2"'),
+                contains('"\\foo\\study2\\sex\\female\\"', '"10 name 1"', '"10 description 1"'),
+                contains('"\\foo\\study2\\sex\\male\\"', '"9 name 2"', '"9 description 2"'),
+                contains('"\\foo\\study2\\sex\\male\\"', '"9 name 1"', '"9 description 1"'),
+        )
```

Comment:
The tests are accumulating a lot of assertions. It would be better that each test tested a specific functionality, in this case, data and metadata.